packages/flutter/test/rendering/independent_layout_test.dart

```diff
@@ -44,7 +46,7 @@ void main() {
     expect(offscreen.child.hasSize, isFalse);
     expect(offscreen.painted, isFalse);
     // Attach the offscreen to a custom render view and owner
-    final RenderView renderView = RenderView(configuration: testConfiguration, window: RendererBinding.instance.window);
+    final RenderView renderView = RenderView(configuration: testConfiguration, window: FakeFlutterView());
```

Comment:
Why are we not using something like `RendererBinding.instance.platformDispatcher.views.single` here? It's longer, but it doesn't require a new implementation of `FlutterView`.

There are a few places that you've added a `FakeFlutterView`, and I'd suggest the same change for all of them.

Reply:
I liked that this makes it explicit that the window that's getting passed in here is completely irrelevant for the test. It's just passed in because it is a required parameter, but nothing does anything with it. If we could use mockito, I'd pass in a mock. But yeah, `RendererBinding.instance.platformDispatcher.views.single` would totally work here. I can change it if you feel strongly about it.

Reply:
I think we should go with the view from the `RendererBinding` for a few reasons:

* That's what was there previously, and we're currently just trying to move to the new API
* I think adding the "it doesn't call anything on the view" check with FakeFlutterView detracts from what's being tested.
    * If something gets added that _does_ reference the view, but not in a way that violates the purpose of the test, this test would fail and be a false negative
* Related to the previous: It feels like you have to know too much about RenderView to read/write/maintain these tests if they fail in the future because of the fake view. You'd have to know that:
    * FakeFlutterView doesn't actually implement anything
    * RenderView didn't previously make use of it for this test
    * Your change made RenderView access the view
    * Finally, you'd have to determine whether or not the restriction on using the view was intended or not, and whether you should be breaking that intention

Reply:
Done.

Reply:
Danke!